src/System.Runtime.Extensions/tests/System/IO/Path.Combine.cs

```diff
@@ -154,25 +161,60 @@ public static void ContainsInvalidCharWithoutRootedAfterArgumentNull_Windows()
             CommonCasesException<ArgumentException>("ab\tcd");
         }
 
-        [ActiveIssue(25665)]
         [Fact]
-        public static void ContainsInvalidCharWithRootedAfterArgumentNull()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public static void ContainsInvalidCharWithoutRootedAfterArgumentNull_Windows_Core()
```

Comment:
What does the name of this test mean?  What's the "ArgumentNull" part?

Reply:
@JeremyKuhne  did you know the reason behind this ?